Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.yml

```
@@ -0,0 +1,49 @@
+category: Forensics & Malware Analysis
+commonfields:
+  id: WildFire-Reports
+  version: -1
+configuration:
+- defaultvalue: https://wildfire.paloaltonetworks.com/publicapi
+  display: Server base URL (e.g., https://192.168.0.1/publicapi)
+  name: server
+  required: true
+  type: 0
+- display: API Key
+  name: token
+  required: false
+  type: 4
+  fromlicense: encrypted
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Generates a Palo Alto Networks WildFire PDF report.
+display: Palo Alto Networks WildFire Reports
+name: WildFire-Reports
+defaultEnabled: true
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: SHA256 hash to check
+      isArray: false
+      name: sha256
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieves results for a file hash using WildFire.
+    execution: false
+    name: wildfire-get-report
```

per the demo. @BenJoParadise FYI.
```suggestion
    name: internal-wildfire-get-report
```